test/data/build_buildpacks-v3_php_cr.yaml

```diff
@@ -5,9 +5,8 @@ metadata:
   name: buildpack-php-build
 spec:
   source:
-    url: https://github.com/cloudfoundry/cf-acceptance-tests
-    contextDir: assets/php
-    revision: main
+    url: https://github.com/shipwright-io/sample-php
```

Comment:
I suggest we keep using the source in repo `https://github.com/cloudfoundry/cf-acceptance-tests` for buildpack tests, so that we can get updating source or dependency version.

Like the `Ruby` bundle problem which we found before, if we keep in our org, it means we cannot find the updated problem in future

Reply:
I think the contract of having our own samples is not only to have full control on them but also implies a maintenance effort.

Reply:
How to do the maintenance? automation or manual?

If the cats test changes today and it is a breaking change, how we can sync and notice that then fail our test soon?

Reply:
Let me provide more context why I think this is safe to do. If we speak only about the repositories we currently use from cf-acceptance-tests, these are the changes in this PR:

| Runtime   | Current Repo      | New Repo |
| ----------- | ----------- | ----------- |
| Go      | [catnip](https://github.com/cloudfoundry/cf-acceptance-tests/tree/develop/assets/catnip)      | https://github.com/shipwright-io/sample-go       |
| Java      | [java](https://github.com/cloudfoundry/cf-acceptance-tests/tree/develop/assets/java)      | https://github.com/shipwright-io/sample-java       |
| PHP      | [php](https://github.com/cloudfoundry/cf-acceptance-tests/tree/develop/assets/php)      | https://github.com/shipwright-io/sample-php       |
| Ruby      | [dora](https://github.com/cloudfoundry/cf-acceptance-tests/tree/develop/assets/dora)      | https://github.com/shipwright-io/sample-ruby       |

## Go

We move to a very minimal [main.go](https://github.com/shipwright-io/sample-go/blob/main/source-build/main.go), compared to what they have in [catnip](https://github.com/cloudfoundry/cf-acceptance-tests/tree/develop/assets/catnip), this should allow us to build the container image faster, therefore faster e2e tests. Do you have something in particular you think we will lose when moving out of catnip? e.g. they use an old go version https://github.com/cloudfoundry/cf-acceptance-tests/blob/develop/assets/catnip/go.mod#L3


## Java

Not sure on this one, maybe @SaschaSchwarze0 can help to understand if there is any disadvantage on moving out from https://github.com/cloudfoundry/cf-acceptance-tests/tree/develop/assets/java

## PHP

I dont see any reasons why not to do this, we are moving out from https://github.com/cloudfoundry/cf-acceptance-tests/blob/develop/assets/php/htdocs/index.php into https://github.com/shipwright-io/sample-php/blob/main/source-build/main.php.


## Ruby

I dont see any reasons here why not to move to a new source code. Do you have any particular concern on this runtime, e.g. dependencies management or similar? The advantage of doing this here is to get a smallest footprint, therefore faster compilation times.

Reply:
For Dockerfile build, I think it is fine to store in our org because we or the user define the Dockerfile, and from our experience before, The Dockerfile can be used forever without any change because all are defined in the Dockerfile and we don't see too much change or problem before.

But for buildpacks build, I have some concern because they depend on the **runtime** and **dependency** versions.
For Java, maybe we upgrade to the latest JDK runtime version, but we don't change the source code, and we are not the expert of the runtime, so maybe they add more function in the source, but we don't know.

For example, the problem which we faced before: https://github.com/cloudfoundry/cf-acceptance-tests/commit/ca812e32764aaabad75553d6f08bb484885a2797

So if the cf-acceptance-tests repo can help keep updating or maintain the test, I think we can reduce our effort for these buildpacks source maintenance and new version problem detection.

But for sure, it is not a blocker of this PR, if you think keeping all in our repo is better, then I am fine to go ahead.

Reply:
Yes is a good point, I think the only one that is a concern in my side is Java. Please see my runtimes examples from above, you will find that the source code is minimal and that having the proper dependencies in place should be a minor effort. @SaschaSchwarze0 whats your take on the Java one?

Reply:
We had some discussion yesterday, because the all samples are very simple and no too much dependency.

I am fine to keep in our repo.